app/javascript/packs/sidebar.js

```diff
@@ -0,0 +1,26 @@
+document.addEventListener('turbolinks:load', function(){
+  var sidebar = document.getElementById("sidebar");
+  var close = document.querySelector(".sidebar-close");
+  var menu = document.getElementById("menu");
+  
+  function openNav() {
+    sidebar.classList.remove('hidden');
+  }
+  
+  function closeNav() {
+    sidebar.classList.add('hidden');
+  }
+
+  function isClose(){
+    sidebar.classList.contains('hidden');
+  }
+  
+  menu.addEventListener('click',openNav);
+
+  close.addEventListener('click',closeNav)
+
+  document.addEventListener('keydown', e => {
```

Comment:
the "burger" menu also works on desktop?, if only works on mobile don't need close menu with the keyboard

Reply:
The burger menu only works on desktop.